mdpow/tests/test_analysis_alchemlyb.py

```diff
@@ -65,10 +61,9 @@ def fep_benzene_directory(tmpdir_factory):
 class TestAnalyze(object):
     def get_Gsolv(self, pth):
         gsolv = pth.join("FEP", "water", "Gsolv.fep")
-        # Loading only works with magic code in restart.Journalled.load()
-        # so that Pickles produced with Python 2 can also be read with 3:
-        G = mdpow.fep.Ghyd(filename=gsolv.strpath)
-        # patch paths
+        with open(gsolv, 'rb') as f:
+            # Needed to read python 2 pickles.
+            G = pickle.load(f, encoding='latin1')
```

Comment:
```suggestion
        # Loading only works with magic code in restart.Journalled.load()
        # so that Pickles produced with Python 2 can also be read with 3:
        G = mdpow.fep.Ghyd(filename=gsolv.strpath)
        # patch paths
```